Assets/Decontainment/Assembly/InstructionMaps.cs

```diff
@@ -17,6 +17,7 @@ public static class InstructionMaps
             {OpCode.BLE, "Branch if less than or equal"},
             {OpCode.BGT, "Branch if greater than"},
             {OpCode.BGE, "Branch if greater than or equal"},
+            {OpCode.BCH, "Branch arg * 1 times out of a hundred"},
```

Comment:
This probably ought to be something like "Branch randomly" while the arg description goes in the arg spec. If we go with "Branch randomly" as the description, BRN would make more sense as the opcode.